### ISSUE TITLE
Wiggly screens rendering issue.  

### ISSUE THREAD

**Issue by chabiss** [c0pperdragon/C64-Video-Enhancement#49]
Finally installed my board from VGP inside 250407 REV C board using a 6569R5 VIC chip.  The results are wiggly. See the video below:

https://youtu.be/WhkywaEn0hI

and a shot picture

![IMG_0576](https://user-images.githubusercontent.com/14151258/96183964-9d5b2900-0eec-11eb-8716-caa1dd4e37bb.JPG)

I've tried different monitors and they all have the same problem.  I've noticed also that some of the C64 demos (like Lunatico) produce some very weird artifacts. Since the board is on 2.6, I assume that I'll need to get to 2.9 to see any improvement of Sprites rendering timing? 

But that wiggly rendering on component output is weird.  Note that the 64 S-Video coming out of the board is fine.  Only the component output has this issue. 

Any idea what could cause this?  Note that this is a 250407 REV C NTSC board that has been converted into PAL by changing the crystal. The machine runs fine in PAL, but the AC 9 volts from the power supply will still come into the machine at 60 HZ.  I'm wondering if this has anything to do with the issue I'm seeing?

**Comment by chabiss** [comment]
Hmmm... This could very well be the same issue as https://github.com/c0pperdragon/C64-Video-Enhancement/issues/36

**Comment by laubzega** [comment]
Unlikely. The screen with #36 is completely unreadable and skewed to one side. Here is just minor wiggling left and right.

**Comment by chabiss** [comment]
@laubzega  you need to watch the video at
https://drive.google.com/file/d/1gpyFiY_BxD6BCISTT9gS2czUl1HwVugZ/view
, not the picture.  The symptoms are very similar to mine.

**Comment by laubzega** [comment]
Then indeed. Apparently #36 is about two distinct issues.

**Comment by c0pperdragon** [comment]
Yes, the second problem posted into this issue was completely unrelated to the original post. I have now removed those entries to not confuse the matter any longer.

If you have tried the 47pF treatment and it did not improve, I still have a second idea. Your board revision seems to have a clock circuit made from individual chips instead of the later integrated solution. This is known to cause a lot of troubles. 
Somes of the problems were fixed with later firmware revisions - especially sprite rendering.

If nothing of this helps, you can also use a special feature of the FPGA board that produces a pixel clock you could feed into the VIC-II to bypass the shaky clock generator - this is explained in more detail in https://github.com/c0pperdragon/C64-Video-Enhancement/issues/38  - but it requires a bit of rework of the C64 main board.

**Comment by chabiss** [comment]
Yep... I fully read https://github.com/c0pperdragon/C64-Video-Enhancement/issues/38 and I'm 99.9% sure I'm hit with the same sparkly pixels issue :) It's going to be a fun project to diagnose. But first I'll move the firmware to 2.9

**Comment by chabiss** [comment]
I moved my firmware to 2.9 and did the pixel clock mod.   

![componentMod](https://user-images.githubusercontent.com/14151258/96842239-a32b9f80-1401-11eb-9830-37a8cf73488d.jpg)

The mod did the fix for me. No more sparkling pixel! I threw everything at it: Sprite Crunch, VSP. It's rock-solid!!! I'm blown away by the board!!!!

**Comment by modeler** [comment]
I had this same issue with my 250407 Rev. B board with 6569R1 (ceramic VIC-II):
 
![animation](https://github.com/c0pperdragon/C64-Video-Enhancement/assets/3359889/b42eb181-9593-424a-8a87-3dba13b71325)

It was especially important (for me at least) to get this RGB mod working with these older boards, as the analogue video output from these is often terrible. The same mod (100 ohm resistor and jumper wire from R24 to FB17) worked like a charm. All of the information I needed to fix this was in this thread. Thank you all so much for sharing this info.
